ts/features/pn/components/MessageDetails.tsx

```diff
@@ -1,193 +1,27 @@
-import React, { useCallback, createRef, useRef } from "react";
-import { pipe } from "fp-ts/lib/function";
-import * as O from "fp-ts/lib/Option";
-import * as RA from "fp-ts/lib/ReadonlyArray";
-import * as SEP from "fp-ts/lib/Separated";
-import { View } from "react-native";
+import React from "react";
 import { ScrollView } from "react-native-gesture-handler";
```

Comment:
Is this import correct or should we use the react-native one?

Reply:
Applied https://github.com/pagopa/io-app/pull/5431/commits/8eb6bb8ecaad487899b7ff08094c09e55a552b60